README.md

```diff
@@ -1,3 +1,84 @@
 # LOCK API
 
-API for LOCK.space international staking service
\ No newline at end of file
+API for LOCK.space international staking service
+
+# Documentation
+
+- [Registration](#registration)
+- [KYC](#kyc)
+- [Staking](#staking)
+- [Deposit](#deposit)
+- [Withdrawal](#withdrawal)
+- [Analytics](#analytics)
+- [Voting](#voting)
+- [Integration Example](#integration-example)
+
+# Registration
+
+1. Contact [support](support@lock.space) to registrate your wallet name
+2. Create an address on selected blockchain (e.g. DeFiChain)
+3. Get and sign message from sign-message endpoint: https://dev.api.lock.space/swagger/#/Authentication/AuthController_getSignMessage
```

Comment:
Get the sign message from sign-message endpoint and sign it with the corresponding private key